{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2020-08-10",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "",
        "Fixer": "mufeedvh"
    },
    "Package": {
        "Registry": "pip",
        "Name": "conference-scheduler-cli",
        "URL": "https://pypi.org/project/conference-scheduler-cli",
        "Downloads": "8538"
    },
    "CWEs": [
        {
            "ID": "CWE-94",
            "Description": ""
        }
    ],
    "CVSS": {
        "Version": "3.1",
        "AV": "L",
        "AC": "L",
        "PR": "N",
        "UI": "R",
        "S": "U",
        "C": "H",
        "I": "H",
        "A": "H",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "7.8"
    },
    "CVEs": [
        "CVE-2018-14572"
    ],
    "Repository": {
        "URL": "https://github.com/pyconuk/conferencescheduler-cli",
        "Codebase": [
            "Python"
        ],
        "Owner": "pyconuk",
        "Name": "conferencescheduler-cli",
        "Forks": 7,
        "Stars": 0
    },
    "Permalinks": [],
    "References": [
        {
            "Description": "GitHub Issue",
            "URL": "https://github.com/PyconUK/ConferenceScheduler-cli/issues/19"
        }
    ]
}